src/LogFileReader.cpp

```diff
@@ -13,13 +14,19 @@ LogFileReader::~LogFileReader()
 {
 }
 
+namespace
+{
+       QString STRING_TO_QTIME = "hh:mm:ss.zzz";
+       QString NEXT_COLUMN = ",";
```

Comment:
Actually constants should be named with all capitals. Using Bill's name, it should be `BATDATA_DELIMITER`